src/osemgrep/cli_interactive/Framed_zipper.mli

```diff
@@ -52,3 +57,5 @@ val is_empty : 'a t -> bool
 val length : 'a t -> int
 val absolute_position : 'a t -> int
 val relative_position : 'a t -> int
+val is_leftmost : 'a t -> bool
```

Comment:
are you talking about the pointer or the frame? the pointer is at the beginning of the list, or the frame (whatever its size) is at the start?

Reply:
left a comment explaining